app/enterprise/0.33-x/developer-portal/best-practice.md

```diff
@@ -6,66 +6,78 @@ chapter: 12
 
-If you are using AWS for you hosting, I would provision a [t2.large](https://aws.amazon.com/ec2/instance-types/t2/) as a starting place. This is a solid base and should be able to handle our expectations without any issue. If you use Google Cloud (GCE), I would suggest starting with a [n1-standard-2](https://cloud.google.com/compute/pricing#standard_machine_types) and if you are using the Azure cloud, I would suggest starting with a [Standard_D2_v3](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-sup1sup).
+If you are using AWS for you hosting, a [t2.large](https://aws.amazon.com/ec2/instance-types/t2/) is a starting place. This is a solid base and should be able to handle expectations without any issue. If you use Google Cloud (GCE), try starting with a [n1-standard-2](https://cloud.google.com/compute/pricing#standard_machine_types) and if you are using the Azure cloud, start with a [Standard_D2_v3](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general#dv3-series-sup1sup).
 
 If you look at those recommendations, you'll see they all have similar starts:
   - At least 2 CPUs
   - Around 8 GiB System memory
 
-I also recommend having this Kong instance be used to only serve the dev portal, and not do any Kong administrative work. To do this, in your kong.conf file, set the following values
+If possible, it's best practice to have this Kong instance be used to only serve the dev portal, and not do any Kong administrative work. To do this, in your kong.conf file, set the following values
 ```
 admin_listen=off
 portal=on
 ```
 We will also want to ensure all other instances of Kong have the portal boolean set to off. This ensures your portal is only being served by a single Kong instance.
 
-## <a name=css></a>Customizing The Portal Design with CSS
+## Customizing The Portal Design with CSS
 
-The Developer Portal has two files that contain CSS and are used to control the design. The main CSS file is `unauthenticated/theme-css` which contains the bulk of the CSS styles for the default layout. The second file is `unauthenticated/custom-css` where owners add and edit their own custom css. Both of these files can be accessed by non authenticated pages. You can access the custom-css file by using the Admin API<br />
-`curl http://localhost:8001/files/unauthenticated/custom-css`<br /><br />
+The Developer Portal has two files that contain CSS and are used to control the design. The main CSS file is `unauthenticated/theme-css` which contains the bulk of the CSS styles for the default layout. The second file is `unauthenticated/custom-css` where owners add and edit their own custom css. Both of these files can be accessed by non authenticated pages. You can access the custom-css file by using the Admin API
+```
+curl http://localhost:8001/files/unauthenticated/custom-css
+```
 
```

Comment:
This section seems mostly duplication of https://docs.konghq.com/enterprise/0.33-x/developer-portal/customization/#customizing-the-kong-developer-portal - I wonder if you could delete the content from that document, and link from there to here instead?